alea/model.py

```diff
@@ -95,16 +95,19 @@ def __init__(
         self._confidence_interval_kind = confidence_interval_kind
         self.confidence_interval_threshold = confidence_interval_threshold
         self.asymptotic_dof = asymptotic_dof
-        self._define_parameters(parameter_definition)
+        nominal_values = kwargs.get("nominal_values", {})
+        self._define_parameters(parameter_definition, nominal_values)
 
         self._check_ll_and_generate_data_signature()
-        self.set_nominal_values(**kwargs.get("nominal_values", {}))
```

Comment:
Is the nearby functionality different from simply using `set_nominal_values`? @hammannr

Reply:
I had to change it since otherwise the static parameters would complain if the nominal value were updated in this step

Reply:
So the outcome is the same. It only had to be changed to accommodate the static parameter functionality

Reply:
Then maybe `set_nominal_values` is not checked by pytest. Would you add a test to increase the coverage?

Reply:
Actually, now it's no longer in use so I would suggest to simply trash the function